accepted/0000-template.md

```diff
@@ -0,0 +1,36 @@
+# {{TITLE: a human-readable title for this RFC!}}
```

Comment:
@thitch97 Why is this file added/changed?  I’d expect that, given the contents of `main`, that this file should be unchanged.

Reply:
I removed the template file by mistake in my original commit and I was attempting to rectify that. I've fixed the issue now.